osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -0,0 +1,205 @@
+            return False
+
+    def download_result_files(self, extraction_order):
+        """
+        Downloads the result files if the conversion was finished,
+        stores the files into the private storage and attaches them as output files to the extraction order
+
+        Args:
+            extraction_order: an ExtractionOrder object to attach the output files
+
+        Returns:
+            True if the job status was fetched successful
+            False if it failed
+        """
+        job_status = self.job_status(extraction_order)
+
+        if job_status and job_status['status'] == 'done' and job_status['progress'] == 'successful':
+            for download_file in job_status['gis_formats']:
+                if download_file['progress'] == 'successful':
+                    result_response = requests.get(download_file['result_url'], headers=self.headers)
+                    output_file = OutputFile.objects.create(
+                        mime_type='application/zip',
+                        file_extension='zip',
+                        content_type=download_file['format'],
+                        extraction_order=extraction_order
+                    )
+                    file_name = str(output_file.public_identifier) + '.zip'
+                    output_file.file = private_storage.save(file_name, ContentFile(result_response.content))
+                    output_file.save()
+            return True
+        else:
+            return False
+
+    def job_status(self, extraction_order):
+        """
+        Get the status of the conversion job
+
+        Args:
+            extraction_order: an ExtractionOrder object containing a process id
+
+        Returns:
+            A status dict on success like:
+                {
+                    "rq_job_id": "4b529c79-559c-4730-9cd2-03ea91c9a5ef",
+                    "status": "done",
+                    "progress": "successful",
+                    "gis_formats": [
+                        {
+                            "format": "fgdb",
+                            "progress": "successful",
+                            "result_url": "http://localhost:8000/api/gis_format/11/download_result/"
+                        },
+                        {
+                            "format": "spatialite",
+                            "progress": "successful",
+                            "result_url": "http://localhost:8000/api/gis_format/12/download_result/"
+                        }
+                    ]
+                }
+            False on error
+        """
+        if not self.is_logged_in:
+            raise Exception('Not logged in for request')
+        request_url = self._create_url(
+            self.api_paths['job']['status'].replace('{rq_job_id}', extraction_order.process_id)
+        )
+        response = requests.get(request_url, headers=self.headers)
+        if not response.status_code == 200 or not response.json():
```

Comment:
seems very similar to other checks above.
see comment [above](https://github.com/geometalab/osmaxx/pull/259/files?diff=unified#r44757566)